### PR TITLE
feat(scraper): Aggregate requests by category

### LINKED ISSUE #6: Details Requests in scraper
Also find a way to apply it to:
- CLI
- API

### PR DESCRIPTION
**Pull request by vvatelot**
Closes #6

### CODE REVIEW

**Comment by vvatelot** [comment]
@CodiumAI-Agent /describe

**Comment by vvatelot** [comment]
@CodiumAI-Agent /review

**Comment by vvatelot** [comment]
@CodiumAI-Agent /improve

File: components/ecoindex/models/scraper.py [hunk 1]
```diff
@@ -2,13 +2,40 @@
...
+
+class MimetypeAggregation(BaseModel):
+    css: MimetypeMetrics = MimetypeMetrics()
+    font: MimetypeMetrics = MimetypeMetrics()
+    html: MimetypeMetrics = MimetypeMetrics()
+    image: MimetypeMetrics = MimetypeMetrics()
+    javascript: MimetypeMetrics = MimetypeMetrics()
+    other: MimetypeMetrics = MimetypeMetrics()
+    video: MimetypeMetrics = MimetypeMetrics()
+
```
Comment (PaulPHPE) [inline components/ecoindex/models/scraper.py:26]: We could add an "audio" type here?

Comment (vvatelot) [reply components/ecoindex/models/scraper.py:26]: Done

**Pull request merged**